charts/brig/templates/deployment.yaml

```diff
@@ -138,11 +138,13 @@ spec:
               scheme: HTTP
               path: /i/status
               port: {{ .Values.service.internalPort }}
+            initialDelaySeconds: 30
```

Comment:
I think your problem would be better solved with a startup probe, see also https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

```
startupProbe:
  httpGet:
   ...
   failureThreshold: 6
   periodSeconds: 5
  ```
 
 That would wait for 30 seconds before moving over to the liveness probe which restarts brig.
 
 Overall, if /i/status fails, however, then this begs the question if the installation works correctly otherwise? If brig doesn't have enough resources likely users will not have adequate latencies either.
 
 Also, could it be that something else regarding networking is not working as it should, rather than this just being a resource problem?

Reply:
Thank you, yes the startupProbe make more sense here, added it.

Yeah, there can be issues due to high network latencies as well, there is a argument "timeoutSeconds", which has default value of 1 second. Maybe, we can increase that as well for these probes.